### PR TITLE
added new css handle for tag component

### PR DESCRIPTION
**Pull request by felipeireslan**
#### What is the purpose of this pull request?

Add a css handle on `Tag` component to be able styling from store-theme.

#### What problem is this solving?

Adding this class we will be able to style Tag component through VTEX store-theme using css selectors.

[Running workspace](https://minicartcoupon--outletespacohering.myvtex.com/)

#### How should this be manually tested?

Add a product your cart, open minicart and add a coupon. After that you will see a Tag component used by [Coupon](https://github.com/vtex-apps/checkout-coupon) styled.

#### Screenshots or example usage

Example: https://prnt.sc/vm5ipo

#### Types of changes

- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- added `vtex-tag` css in order to use css handles on store-theme.
```
Comment (klzns) [inline CHANGELOG.md:12]:
```suggestion
- Class `vtex-tag` on `Tag` component.
```

**Review by klzns — CHANGES_REQUESTED** [review]
Thanks for the pull request! Just a minor correction in the changelog

File: CHANGELOG.md (hunk 1 above)
Comment (felipeireslan) [reply CHANGELOG.md]: @klzns done.

**Comment by felipeireslan** [comment]
@klzns done.

**Review by klzns — APPROVED** [review]

**Review by matheusps — APPROVED** [review]
LGTM

**Comment by estacioneto** [comment]
Thank you for your contribution, @felipeireslan!
Since we're in a pretty critical moment (just before Black Friday), I'll release your PR and deploy it next week, ok? :)

**Comment by estacioneto** [comment]
@felipeireslan can you please resolve the `CHANGELOG.md` conflicts? When you're done, we can release a new version :)

**Comment by felipeireslan** [comment]
@estacioneto done. Can you please check again ?

**Comment by estacioneto** [comment]
@felipeireslan just merged and release a new version here: #1355 . Will be deploying soon 🚀 
Thanks again for the contribution!

**Pull request merged**